readers-guide.md

```diff
@@ -11,6 +11,10 @@ It is common for individuals to have duties which span multiple roles.
 
 Below is a brief explanation of each of these sections and how they are used within the criteria of the Standard.
 
+## Preamble
```

Comment:
```suggestion
## Introduction
```

'Preamble' is a less common word than 'introduction', and carries more (unnecessary) overtones of legal jargon.